tests/pdk/finfet_pdk/test_digital.py

```diff
@@ -16,6 +16,7 @@ def test_dig_1():
     name = f'ckt_{get_test_id()}'
     netlist = textwrap.dedent(f"""\
     .subckt dig22inv a o vccx vssx
+    * @: Generator(pattern='CS')
```

Comment:
What is this for?

Reply:
Since I am removing the generators file I do not have a method to query from PDK if a generator exists for a subcircuit or not. In the current method, all basic/user template subcircuits which can be generated using primitive generators have this generator constraint. This was an outlier and thus I added this constraint in the netlist. Without this, I was not able to fix the test as the elements in dig22sing had nf=1.

Let me know if there is any other method to query from the PDK then I can use that to generate this constraint automatically instead of being defined in the netlist.

Reply:
There is already a way to query PDK and we are using it in various places. Please search for get_generator(). Examples:
generate_primitive_lef.py: `elif name == 'generic' or get_generator(name.lower(), pdk_dir):`
test_digital.py/test_dig22inv: `primitive_generator = main.get_generator(name, pdk_dir)`

Reply:
By the way, this could be the first check for any level of hierarchy before descending in. Example: Existing layout.

Reply:
Is there any way we can make the generator query case insensitive?

Reply:
Go with lowercase.

Reply:
Please check the current changes, if it looks fine for you?